src/main/java/net/mcreator/element/types/RangedItem.java

```diff
@@ -49,6 +49,8 @@
 	public int stackSize;
 	public MItemBlock ammoItem;
 	public boolean shootConstantly;
+	public boolean preserveShots;
```

Comment:
We call them projectiles, not shots. I would recommend renaming shots to projectiles, here, in texts and in .mc help texts so we keep the unified terminology